OfflineMapAreasExample/OfflineMapAreasExample/OfflineMapAreasExampleView.swift

```diff
@@ -34,7 +37,10 @@ struct OfflineMapAreasExampleView: View {
                 }
             }
             .sheet(isPresented: $isShowingOfflineMapAreasView) {
-                OfflineMapAreasView(map: map)
+                OfflineMapAreasView(
+                    online: onlineMap,
+                    selection: $selectedMap
+                )
```

Comment:
Combined, the line would be shorter than 80 columns, if you want to.

```suggestion
                OfflineMapAreasView(online: onlineMap, selection: $selectedMap)
```